docs/engineering-feedback/feedback-faq.md

```diff
@@ -6,34 +6,34 @@ The questions below are common questions related to the feedback process. The an
+Submitting to ISE Feedback allows the ISE Feedback team to coalesce feedback across a wide range of sources, and thus create a unified case to submit to the appropriate Azure engineering team(s).
 
 ## How can a customer track the status of a specific feedback item?
 
-At this time, customers are not able to directly track the status of feedback submitted via CSE Feedback.  The CSE Feedback process is internal to Microsoft, and as such, available only to Microsoft employees.  Customers may request an update from their CSE engineering partner or Microsoft account representative(s).
+At this time, customers are not able to directly track the status of feedback submitted via ISE Feedback.  The ISE Feedback process is internal to Microsoft, and as such, available only to Microsoft employees.  Customers may request an update from their ISE engineering partner or Microsoft account representative(s).
 
-Customers can also submit their feedback directly via GitHub or UserVoice (as appropriate for the specific service), and inform their CSE engineering partner.  The CSE engineer should submit the feedback via the CSE Feedback process, and in doing so reference the previously created issue.  Customers can follow the GitHub or UserVoice item to be alerted on updates.
+Customers can also submit their feedback directly via GitHub or UserVoice (as appropriate for the specific service), and inform their ISE engineering partner.  The ISE engineer should submit the feedback via the ISE Feedback process, and in doing so reference the previously created issue.  Customers can follow the GitHub or UserVoice item to be alerted on updates.
 
 ## How can a Microsoft employee track the status of a specific feedback item?
 
-The easiest way for a Microsoft employee within CSE to track a specific feedback item is to [follow the feedback (a work item)](https://learn.microsoft.com/azure/devops/boards/work-items/follow-work-items?view=azure-devops) in Azure DevOps.
+The easiest way for a Microsoft employee within ISE to track a specific feedback item is to [follow the feedback (a work item)](https://learn.microsoft.com/azure/devops/boards/work-items/follow-work-items?view=azure-devops) in Azure DevOps.
 
-## As a Microsoft employee within CSE, if I submit a feedback and move to another dev crew engagement, how would my customer get an update on that feedback?
+## As a Microsoft employee within ISE, if I submit a feedback and move to another dev crew engagement, how would my customer get an update on that feedback?
 
 If the feedback is also submitted via GitHub or UserVoice, the customer may elect to follow that item for publicly available updates.  The customer may also contact their Microsoft account representative to request an update.
 
-## As a Microsoft employee within CSE, what should I expect/do after submitting feedback via CSE Feedback?
+## As a Microsoft employee within ISE, what should I expect/do after submitting feedback via ISE Feedback?
```

Comment:
After reading a bit more - I think we can revert all the sections of CSE feedback to CSE Feedback, that is still the name of the tool (CSE being a part of ISE) -- but also add a backlog item to eventually remove the sections on CSE feedback from the playbook (unless you want to do it now) ... This is an internal process, that has now changed to doing feedback through the SA teams, and this was added to the playbook at some point when we had a different view of the playbook.  Currently, the rule is "all internal processes should be in the wiki and not in the playbook"